Tests/js/views/ez-studiopresentationview.html

```diff
@@ -27,13 +27,17 @@
         filter: loaderFilter,
         modules: {
             "ez-studiopresentationview": {
-                requires: ['ez-templatebasedview'],
+                requires: ['ez-templatebasedview', 'ez-height-fit'],
                 fullpath: "../../../Resources/public/js/views/ez-studiopresentationview.js"
             },
             "ez-templatebasedview": {
                 requires: ['ez-view', 'handlebars', 'template'],
                 fullpath: "../../../Resources/public/js/views/ez-templatebasedview.js"
             },
+            "ez-height-fit": {
+                requires: ['ez-view'],
```

Comment:
AFAICS, ez-height-fit actually depends on the module `view` not `ez-view`. I think you also need to add `node-screen` (because of the usage of `getY`) and `node-style` (because of the usage of `setStyle`)